tools/cldr-apps/src/main/webapp/js/special/vetting_participation.js

```diff
@@ -0,0 +1,208 @@
+'use strict';
+
+/**
+ * Based on specials/forum.js
+ *
+ * TODO: use modern JavaScript -- import/export, not define
+ * TODO: avoid dependencies on legacy code
+ * TODO: put code for progress bar, etc., in a shared module, share with, e.g., vsummary.js
+ */
+define("js/special/vetting_participation.js", ["js/special/SpecialPage.js"], function(SpecialPage) {
+	var _super;
+	
+	'use strict';
+
+	const tableId = "vettingParticipationTable";
+	const fileName = "vettingParticipation.csv";
+	const onclick = "cldrStCsvFromTable.downloadCsv("
+		+ "\"" + tableId + "\""
+		+ ", "
+		+ "\"" + fileName + "\""
+		+ ")";
+
+
+
+	function Page() {
+		// constructor
+	}
+	
+	/**
+	 * Fetch the Forum Participation data from the server, and "load" it
+	 *
+	 * @param params an object with various properties; see SpecialPage.js
+	 */
+	function show(params) {
```

Comment:
show() is over 100 lines, way more than one screenful -- could improve with subroutines

Reply:
fixed